File: packages/expo-constants/android/src/main/java/expo/modules/constants/ConstantsPackage.java
package expo.modules.constants;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.modules.core.ExportedModule;
import expo.modules.core.BasePackage;
import expo.modules.core.interfaces.InternalModule;

public class ConstantsPackage extends BasePackage {
  @Override
  public List<InternalModule> createInternalModules(Context context) {
    return Collections.singletonList((InternalModule) new ConstantsService(context));
  }

  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ConstantsModule(context));
  }
}

File: packages/expo-constants/android/src/main/java/expo/modules/constants/ConstantsPackage.kt
package expo.modules.constants

import android.content.Context

import expo.modules.core.BasePackage
import expo.modules.core.interfaces.InternalModule

class ConstantsPackage : BasePackage() {
  override fun createInternalModules(context: Context): List<InternalModule> =
    listOf(ConstantsService(context))
}
